contributing_to_docs/doc_guidelines.adoc

```diff
@@ -471,53 +511,101 @@ $ lorem.sh
+* Use `[source,terminal]` for `oc` commands or any terminal commands to enable
+syntax highlighting. Any `[source]` metadata must go on the line directly before
+the code block. For example:
 +
 ....
-Lorem ipsum
-
+[source,terminal]
 ----
-$ lorem.sh
+$ oc get nodes
 ----
 ....
++
+If you are also showing a code block for the output of the command, use
+`[source,terminal]` for that code block as well.
+
+* Use source tags for the programming language used in the code block to enable
+syntax highlighting. For example:
 
-* Do not use more than one command per code block. For example, the following must be split up into three separate code blocks.
+** `[source,yaml]`
+** `[source,go]`
+** `[source,javascript]`
+
+* Do not use more than one command per code block. For example, the following must
+be split up into three separate code blocks:
 +
+....
 Run the following commands to create templates you can modify:
-+
+
+[source,terminal]
 ----
 $ oc adm create-login-template > login.html
 ----
-+
+
+[source,terminal]
 ----
 $ oc adm create-provider-selection-template > providers.html
 ----
-+
+
+[source,terminal]
 ----
 $ oc adm create-error-template > errors.html
 ----
-+
+....
 
-* Separate a command and its related output into individual code blocks. For example:
+* Separate a command and its related example output into individual code blocks.
+This allows the command to be easily copied using the button on
++++docs.openshift.com+++.
 +
-Use the `oc new-project` command to create a new project.
+In addition, prepend the code block for the output with the title `.Example output`
+to make it consistently clear across the docs when this is being represented. A
+lead-in sentence explaining the example output is optional. For example:
 +
+....
+Use the `oc new-project` command to create a new project:
+
+[source,terminal]
 ----
 $ oc new-project my-project
 ----
 
-+
-The output verifies that a new project was created.
-+
+The output verifies that a new project was created:
 
+.Example output
+[source,terminal]
 ----
 Now using project "my-project" on server "https://openshift.example.com:6443".
 ----
+....
+
+* To markup command syntax, use the code block and wrap any replaceable values in
+angle brackets (`<>`) with the required command parameter, using underscores
+(`_`) between words as necessary for legibility. For example:
++
+....
+The following command returns a list of objects for the specified object type:
+
+[source,terminal]
+----
+$ oc get <object_type> <object_id>
+----
+....
++
+This renders as:
++
+--
+> The following command returns a list of objects for the specified object type:
+
+> ----
+> $ oc get <object_type> <object_id>
+> ----
+--
++
+NOTE: Avoid using full command syntax inline with sentences.
 
-* Try to use callouts to provide information on what the output represents when required.
+* If you must expand or provide additional information on what a line of a code
```

Comment:
_If you must expand_  Not sure what this means. 

_If you need to provide additional information_ should be fine??

Reply:
Yeah I wasn't exactly sure what it was getting at either. And you're right, so taking it out!